source/api/extension/text-granularity/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Text Granularity Extension
+layout: spec
+tags: [extension, granularity]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [semantic versioning][notes-versioning].
+Changes will be tracked within the document.
+
+{% include copyright.md %}
+
+## Table of Contents
+{:.no_toc}
+
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Introduction
+
+### 1.1 Objectives and Scope
+
+One use of annotation in IIIF is to associate text with IIIF resources, with the text originating from optical character recognition (OCR) software, manual transcription, existing digitized text, or other sources. Furthermore, it is a common practice to produce sets of annotations at a particular level of ‘text granularity’; that is to say that each annotation in the set references the same unit of text, such as a character, line, or page. This extension recommends a pattern for indicating the level of text granularity of an annotation.
```

Comment:
One use of annotations in IIIF is to associate text with IIIF resources, with the text originating from many potential sources, including optical character recognition (OCR) software, manual transcription, and existing digitized text. Furthermore, it is a common practice to produce sets of annotations at a particular level of ‘text granularity’, where each annotation in the set references the same unit of text, such as a character, line, or page. This extension recommends a pattern for indicating the level of text granularity for an annotation.